src/app/wallet/wallet/send/send.component.ts

```diff
@@ -231,6 +231,21 @@ export class SendComponent implements OnInit, OnDestroy {
           }
           break;
       }
+    } else {
+      switch (txType) {
+        case TxType.BLIND:
+          this.send["input"] = TxType.BLIND;
+          this.send["output"] = TxType.BLIND;
+          break;
```

Comment:
Instad just set this.send.input = this.send.output = TxType.BLIND,same for other entries. simpler way is to just do
this.send.input = this.send.output = txType